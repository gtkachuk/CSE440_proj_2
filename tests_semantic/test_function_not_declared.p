program testFunctionNotDeclared;

class testFunctionNotDeclared

BEGIN
   
   VAR
      compilerWorks	      : integer;

FUNCTION setCompilerWorks;
BEGIN
   compilerWorks := 5
END ;

FUNCTION testFunctionNotDeclared;
BEGIN
   compilerWorks := setLightSwitchWorks(0)
END

END
.
