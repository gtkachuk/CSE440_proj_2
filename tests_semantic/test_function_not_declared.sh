#!/bin/sh

. ../tests_lib.inc

test_diff_sem test_function_not_declared.p
