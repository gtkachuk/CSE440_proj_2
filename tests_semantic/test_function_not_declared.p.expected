ERROR:18:Function "setLightSwitchWorks" not declared
Errors detected. Exiting.
